packages/core/stories/toolkit-provider.doc.stories.mdx

```diff
@@ -96,6 +96,129 @@ It requires no configuration, but is included in your app when you use a Toolkit
 AriaAnnouncer is ever required, only the outermost instance of ToolkitProvider will instantiate the AriaAnnouncer. If you use
 nested ToolkitProviders, these will not inject further instances of AriaAnnouncer.
 
+## Breakpoints
+
+The `breakpoints` prop on ToolkitProvider can be used to customize the css media queries that custom hooks like `useBreakpoints` and `useCurrentBreakpoint` are using internally. These hooks are usually used to enable the responsive props, where you can pass multiple values (one per breakpoint) or a single value that would take effect across all different screen sizes. These props can mostly be found in the layout components for now. Look for an example in the [FlexLayout](/docs/documentation-core-layout-flexlayout--page) component documentation.
+
+The default value is the following breakpoint size mapping object.
+
+```js
+{
+  xs: 0,
+  sm: 600,
+  md: 960,
+  lg: 1280,
+  xl: 1920,
+}
+```
+
+## useDensity
+
+You can use `useDensity` hook to access the currently active density of your app in the provided scope. The useDensity hook returns the string value of the currently set density. The possible values are "touch", "low", "medium", and "high". You can optionally override the value by passing your own value as the first parameter to the function.
+
+### API
+
+```js
+import { useDensity } from "@jpmorganchase/uitk-core";
+```
+
+### Example use
+
+```tsx
+import { useDensity } from "@jpmorganchase/uitk-core";
+import cx from "classnames";
+
+function MyComponent() {
+  const [override, setDensityOverride] = useState();
+  const density = useDensity(override);
+  return (
+    <>
+      <div
+        className={cx("myComponent", {
+          "myComponent--touch": density === "touch",
+        })}
+      >
+        Hello World!
+      </div>
+      <button
+        onClick={() => {
+          setOverride("touch");
+        }}
+      >
+        Set Touch Density
+      </button>
+    </>
+  );
+}
+```
+
+## useTheme
+
+You can use the `useTheme` hook to access the value which indicates which themes are currently active. The `useTheme` hook returns the array value of the currently set themes in scope. The available themes by default are "uitk-light" and "uitk-dark". You can optionally override these values by providing an array of theme names to `ToolkitProvider` via the `themes` prop.
+
+```tsx
+import { useTheme } from "@jpmorganchase/uitk-core";
+
+function MyComponent() {
+  const [currentTheme] = useTheme();
+  return (
+    <div>{currentTheme === "uitk-light" ? "Light Theme" : "Dark Theme"}</div>
+  );
+}
+```
+
+### Example use
```

Comment:
Seems like a duplication?

Reply:
Indeed. Not sure what I was trying to do here. Removed